### DISCUSSION TITLE
2024/10/09/vacuous-truth

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#13] [General]
# 2024/10/09/vacuous-truth

來一個我自己覺得滿有趣的 Python 程式問答題。在 Python 裡，all() 這個函數可以接一個串列，串列裡的每一顆元素都成立的時候就會回傳 True，反之只要有一顆不成立，就會是 False，例如：>>> all([True, True, True])True>>> ...

https://kaochenlong.com/2024/10/09/vacuous-truth.html

**Comment by zouyu13** [comment]
........
